kalite/distributed/static/js/distributed/intro-injector.js

```diff
@@ -0,0 +1,122 @@
+//on page load
+$(function() {
+  console.log( "ready!" );
```

Comment:
Eventually you'll want to get rid of this, or if you feel that console output could be helpful then make a more meaningful message.